docs/setup_wsl.md

```diff
@@ -92,10 +92,28 @@ $ whoami
 root  # SOMEthiNG IS WRONG
 ```
 
+We recommend you completely reinstall Ubuntu. (Note this will _delete all of the data associated with Ubuntu_. If you've just installed it, that's fine.)
```

Comment:
Im unable to comment on line 89 directly, so commenting here.

Thoughts on removing the sentence "We recommend that you [uninstall](https://www.makeuseof.com/uninstall-wsl-windows/) and [reinstall](#install-wsl) WSL to try again." in line 89?

Reply:
+1.  Another option would be to replace the link with the docs suggested by @jamesjuett

Reply:
Thanks for pointing this out. I had intended to remove that but missed it! Fixed in 3449ec7